src/formatDate.js

```diff
@@ -50,7 +50,57 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const SEPARATOR_INDEX = 3;
+  const START_ITERATION = 0;
+  const END_ITERATION = 3;
+  const LAST_DIGITS = 30;
+  const CURRENT_CENTURY_YEAR = '20';
+  const PREVIOUS_CENTURY_YEAR = '19';
+  const stringItems = date.split(fromFormat[SEPARATOR_INDEX]);
+  const newDate = [];
+
+  for (let i = START_ITERATION; i < END_ITERATION; i++) {
+    let toFormatItem = toFormat[i];
+
+    switch (true) {
+      case !fromFormat.includes(toFormatItem):
+        const SHORT_YEAR = 'YY';
+        const LONG_YEAR = 'YYYY';
```

Comment:
please also move SHORT_YEAR and LONG_YEAR to the bottom

Reply:
I don’t understand where exactly I need to move it down